Sources/protoc-gen-swift/ExtensionGenerator.swift

```diff
@@ -115,10 +115,13 @@ struct ExtensionGenerator {
         let scope = swiftDeclaringMessageName == nil ? "" : "static "
         let traitsType = descriptor.getTraitsType(context: context)
 
-        var fieldNamePath = self.protoPath
-        if fieldNamePath.hasPrefix(".") {
-            fieldNamePath.remove(at: fieldNamePath.startIndex)
-            fieldNamePath = "\"\(fieldNamePath)\""
```

Comment:
Actually, why do we only quote the string if it started with a '.', seems like if we didn't do that pruning then we don't quote the value here.  Should the `p.print("fieldName: \(fieldNamePath),\n")` always add the quotes instead?